src/utils/generator/ReplicaFiller.cpp

```diff
@@ -37,105 +40,102 @@ using std::endl;
  * the Dummy domain decomposition?
  */
 class ParticleContainerToBasisWrapper : public ParticleContainer {
-public:
-	ParticleContainerToBasisWrapper() = default;
+ public:
+    ParticleContainerToBasisWrapper() = default;
 
-	~ParticleContainerToBasisWrapper() override = default;
+    ~ParticleContainerToBasisWrapper() override = default;
 
-	void readXML(XMLfileUnits& xmlconfig) override {};
+    void readXML(XMLfileUnits& xmlconfig) override {};
 
-	void setBoundingBox(std::shared_ptr<Object> object) { _object = std::move(object); }
+    void setBoundingBox(std::shared_ptr<Object> object) { _object = std::move(object); }
 
-	bool addParticle(Molecule& particle, bool inBoxCheckedAlready = false, bool checkWhetherDuplicate = false,
-					 const bool& rebuildCaches = false) override {
-		double r[3] = {particle.r(0), particle.r(1), particle.r(2)};
-		if(_object && !_object->isInside(r)) {
-			return false;
-		}
-		_basis.addMolecule(particle);
-		return true;
-	}
+    bool addParticle(Molecule& particle, bool inBoxCheckedAlready = false, bool checkWhetherDuplicate = false,
+                     const bool& rebuildCaches = false) override {
+        double r[3] = {particle.r(0), particle.r(1), particle.r(2)};
+        if (_object && !_object->isInside(r)) {
+            return false;
+        }
+        _basis.addMolecule(particle);
+        return true;
+    }
```

Comment:
although I love the change to spaces, ls1 is in tabs and we should strive for consistency :(